examples/text_labels_and_annotations/arrow_demo.py

```diff
@@ -133,7 +133,7 @@ def make_arrow_graph(ax, data, size=4, display='length', shape='right',
         # rotate based on direction of arrow (cos, sin)
         M = [[cos, -sin], [sin, cos]]
         x, y = np.dot(M, orig_positions[where]) + [x_pos, y_pos]
-        label = r'$r_{_{\mathrm{%s}}}$' % (pair,)
+        label = fr'$r_{{_{{\mathrm{{{pair}}}}}}}$'
```

Comment:
as much as I generally prefer f-strings over percent formatting... this one is a little ridiculous as an f-string...

Reply:
Quite strongly agree that tex strings should generally keep using %-formatting.

Reply:
:+1: I reverted all of the double `{}` additions now.